### PR TITLE
update contributing guides to include cla (close #594)

### LINKED ISSUE #594: Include the CLA (Contributor License Agreement) in Contributing Guidelines
Currently, all of Hasura's Contributing or Readme instructions do not mention that the contributor has to sign a CLA (Contributor License Agreement). It may be trivial to some, but not all.

I have tried to fix this in PRs #590, #591, #592, #593 to cover all Contributing docs I am aware of

### PR DESCRIPTION
**Pull request by Bharat123rox**
### Description

What component does this PR affect? 

- [ ] Server
- [ ] Console
- [ ] CLI
- [ ] Docs
- [x] Community Content
- [ ] Build System

Requires changes from other components? If yes, please mark the components:

- [ ] Server
- [ ] Console
- [ ] CLI
- [ ] Docs
- [ ] Community Content
- [ ] Build System

### Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Docs update
- [x] Community content

### Checklist:

- [x] I have read the **[contributing guide](https://github.com/hasura/graphql-engine/blob/master/CONTRIBUTING.md)** and my code conforms to the guidelines.
- [ ] This change requires a change in the documentation. 
- [ ] I have updated the documentation accordingly.
- [ ] I have added required tests.

### CODE REVIEW

**Comment by shahidhk** [comment]
Hello @Bharat123rox. Thanks for the PR. As mentioned in https://github.com/hasura/graphql-engine/issues/594#issuecomment-425851140, please merge #591, #592 and #593 to this PR and then we can merge!

**Comment by Bharat123rox** [comment]
It says merging pull requests need write access to the repository, so I don't know how I can go ahead and do it

**Comment by Bharat123rox** [comment]
You can go ahead and merge each of them as they do not have merge conflicts with each other or with the  `master` branch but since I don't have write access, I don't know how to do it

**Comment by shahidhk** [comment]
@Bharat123rox You cannot do it from the GitHub website. I can help you with doing it using the `git` cli.

Install and setup git by following the instructions here: https://git-scm.com/book/en/v2/Getting-Started-Installing-Git

Once you install git, setup your email and username:

```bash
git config --global user.name "<your-name>"
git config --global user.email "<your-email>"
```

Then clone your fork of GraphQL Engine:

```bash
git clone https://github.com/Bharat123rox/graphql-engine
```

Goto the cloned directory:
```bash
cd graphql-engine
```

Checkout to the `patch-1` branch:
```bash
git checkout patch-1
```

Make your changes by opening and editing files in this folder using your preferred editor.

Once you're done with changes, commit and push those files:
```bash
git commit -am "added cla instructions"
git push
```

Your changes will now appear here. Let me know if you are stuck somewhere.

**Comment by Bharat123rox** [comment]
Is this enough now to merge this PR with the main branch, or is there anything else to be done? In case it is merged, am I eligible for a T-shirt or other goodies from Hasura?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -64,6 +67,8 @@ before-mentioned sections).
 
 - Rebase master with your branch before submitting pull request.
 
+- Any pull request requires the signing of the CLA (Contributor License Agreement) before or after it is 	  submitted, which can be found [here](https://cla-assistant.io/hasura/graphql-engine)
```
Comment (shahidhk) [inline CONTRIBUTING.md]: This can be removed as we have already mentioned in the file above.

**Comment by shahidhk** [comment]
Thanks for the PR @Bharat123rox. We'll get in touch for the goodies and stuff.

**Pull request merged**